modules/programs/neovim.nix

```diff
@@ -24,6 +24,10 @@ let
     merge = mergeOneOption;
   };
 
+  moduleConfigure =
```

Comment:
Can change formatting to
```
  moduleConfigure =
    optionalAttrs (cfg.extraConfig != "") {
      customRC = cfg.extraConfig;
    }
    // optionalAttrs (cfg.plugins != []) {
      packages.home-manager.start = cfg.plugins;
    };
```
to avoid the long lines and to have `//` at the start of the line.